### ISSUE TITLE
Add typescript linting

### ISSUE THREAD

**Issue by prometheas** [prometheas/CommunityAllyWebApp#1]
Add dev dependencies and configs to implement code linting of typescript files.

**Comment by prometheas** [comment]
Hey @taylorclark: presumably, it's the [angular-tslint-rules](https://www.npmjs.com/package/angular-tslint-rules) used by the Angular community that are desired.

**Comment by CommunityAlly** [comment]
Yes, let's go with the most accepted standard. Could I see how the linter's rules would change a single file? website\ngApp\chtn\manager\manage-committees-ctrl.ts, for example.
